### PR TITLE
Fix for caller id number

### PR DESCRIPTION
**Pull request by erikh360**
I copied this into the dockerconfig repo and tested it. Works!

### CODE REVIEW

File: src/voice_missed_call_service.js [hunk 1]
```diff
@@ -38,7 +39,8 @@ go.app = function() {
 
         // EndState
         self.add('state_end', function(name) {
-            return new EndState(name, {text: ''});
+            // freeswitch crashes if the text is empty
+            return new EndState(name, {text: 'Thank you.'});
```
Comment (rudigiesler) [inline src/voice_missed_call_service.js:43]: I think for freeswitch, we can have the text as `.`, which translates to a pause in the text to speech

**Review by rudigiesler — COMMENTED** [review]
Just a small comment, looks good otherwise.

File: src/voice_missed_call_service.js (hunk 1 above)
Comment (erikh360) [reply src/voice_missed_call_service.js:43]: Thanks, let me test that.

Comment (erikh360) [reply src/voice_missed_call_service.js:43]: @rudigiesler That also made freeswitch crash, maybe it only works if it is part of more text. The `Thank you` should be fine, the whole call just lasts 2 seconds.

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**